Update tanka.md with steps for using k8s-alpha lib (#2528)
grafana/loki: docs/sources/installation/tanka.md

jb install github.com/grafana/loki/production/ksonnet/promtail
```

> Note: As of 2020-08-13 we use some as of yet non GA features. This step will likely not be necessary in future tanka releases. Simply install this library which we'll use as an override:

Reviewer: ```suggestion
> **Note:** As of 2020-08-13, we use some features that are not yet generally available. This step will be unnecessary in future Tanka releases. For now, install this library, which we'll use as an override:
```

```bash
jb install github.com/jsonnet-libs/k8s-alpha/1.14
```

Next, override the `lib/k.libsonnet` with the following

```jsonnet
(import 'github.com/jsonnet-libs/k8s-alpha/1.14/main.libsonnet')
+ (import 'github.com/jsonnet-libs/k8s-alpha/1.14/extensions/kausal-shim.libsonnet')
```

> Be sure to replace the username, password and the relevant `htpasswd` contents.

Reviewer: ```suggestion
Be sure to replace the username, password, and the relevant `htpasswd` contents.
```

> Making sure to set the value for username, password, and `htpasswd` properly,

Reviewer: This line seems repetitive.
